types.go

```diff
@@ -102,6 +102,12 @@ func (mt MapType) hasPerCPUValue() bool {
 	return mt == PerCPUHash || mt == PerCPUArray || mt == LRUCPUHash || mt == PerCPUCGroupStorage
 }
 
+// isQueueStack returns true if the Map is a Queue (BPF_MAP_TYPE_QUEUE)
+// or Stack (BPF_MAP_TYPE_STACK)
+func (mt MapType) isQueueStack() bool {
```

Comment:
This is a shortcut for the map's type instead of its semantics. Maybe `needsPop()` would be more descriptive?